sensu-plugins-elasticsearch.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
   s.version                = SensuPluginsElasticsearch::Version::VER_STRING
 
   s.add_runtime_dependency 'rest-client',       '1.8.0'
-  s.add_runtime_dependency 'elasticsearch',     '~> 1.0.14'
+  s.add_runtime_dependency 'elasticsearch',     '>= 1.0.10'
```

Comment:
I'm not seeing a clean way either.

As for the original issue in #48, I suspect it's actually the way the check is being invoked or an environmental issue. I have this exact same version installed and it works fine for me. 
@kaushiksriram100 can you check out my comment at https://github.com/sensu-plugins/sensu-plugins-elasticsearch/issues/48#issuecomment-300688956.